frontend/src/__tests__/unit/utils/time.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { convertPeriodicTimeToSeconds } from '~/utilities/time';
+
+describe('Convert periodic time to seconds', () => {
+  it('should convert hours to seconds', () => {
+    expect(convertPeriodicTimeToSeconds('5Hour')).toBe(5 * 3600);
```

Comment:
Another use-case for just flat out typing a number -- not quite sure what you're after here. Why not `5 * 60 * 60`?